snippets/rust/rust.json

```diff
@@ -384,6 +427,11 @@
         "body": ["type ${1:Alias} = ${2:Type};"],
         "description": "type … = …;"
     },
+    "use": {
```

Comment:
I believe this can be omitted as it's not a complex snippet *and* IIRC rust-analyzer will already support some auto-completion for imports fairly well.